cmd/tsbs_run_queries_timescaledb/main.go

```diff
@@ -204,7 +204,13 @@ func (p *processor) ProcessQuery(q query.Query, isWarm bool) ([]*query.Stat, err
 	} else if p.opts.printResponse {
 		prettyPrintResponse(rows, tq)
 	}
+	// Fetching all the rows to confirm that the query is fully completed.
```

Comment:
we only need to do this in the case of using `pgx` right? Can we guard it with an if statement?